scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SMALL_INSTALL=false
+
+while test $# -gt 0; do
+    case "$1" in
+        -h|--help)
+            echo "quick-helm-install.sh"
```

Comment:
Suggesting to change like this (or similarly)
```suggestion
            echo -e "Usage:\n\tquick-helm-install.sh [options]"
```

Reply:
Okay